packages/frontend/src/components/NewsDrawer/index.tsx

```diff
@@ -46,15 +46,15 @@ export default function NewsDrawer() {
   return (
     <>
       <Button colorScheme="secondary" variant="link" mr={6} onClick={onOpen}>
-        <Text>What's new</Text>
+        <Text textStyle="subhead-1">What's new</Text>
         {localLatestTimestamp !== latestNewsTimestamp && (
           <Box
             borderRadius="50%"
-            bg="#C05050"
+            bg="primary.600"
             w={1.5}
             h={1.5}
```

Comment:
nit (doesn't matter if you put or not)

```suggestion
            boxSize={1.5}
```